code/game/machinery/computer/camera.dm

```diff
@@ -135,7 +135,7 @@
 		return 0
 	set_current(C)
 
-	if(!is_contact_area(get_area(C)))
+	if (!(C.z in GetConnectedZlevels(starting_z_level)))
```

Comment:
This doesn't seem to make much sense, it's supposed to simulate the presence of a network connection, what this seems to do instead is create a quantum tunnel where you can have brought the computer on the other side of the universe and you'd still be able to see the cameras on the other end, just because they were near at the start of the round?

Reply:
you won't be able to open the camera console in the first place, cause of the other check